### DISCUSSION TITLE
Vulnerability reachable in the code

### DISCUSSION THREAD

**Discussion by samigt** [community/community#12363] [Code Security] [answered]
Hello,

Is it on the roadmap to trace a dependency vulnerability to a function in the code ?

Thanks,
Sami

**Comment by erinhav** [comment, marked as answer]
Hi @SamiGT -- this feature just launched in public beta for Python alerts. Learn more [here](https://github.blog/2022-04-14-dependabot-alerts-now-surface-if-code-is-calling-vulnerability/)!

**Reply by samigt** [reply]
Thank you @erinhav !!

Yes I was very happy to see the Python announcement.

I could not find an item for Javascript on the roadmap, any idea when to expect the same feature for Javascript, Node.js, Front end and back end ?

Regards,
Sami

**Reply by erinhav** [reply]
Hi Sami, I can't give an exact date yet, but JavaScript is likely next for the feature. As that shapes up, we'll add it to the roadmap

**Reply by samigt** [reply]
Thank you
